Apps/Sandcastle/gallery/Particle System Tails.html

```diff
@@ -0,0 +1,260 @@
+
+// options
+var rocketOptions = {
+    numberOfParticles : 50.0,
+    iterOffset :  0.1,
+    cartographicStep : 0.000001,
+    baseRadius : 0.0005,
+
+    colorOptions : [{
+        minimumRed : 1.0,
+        green : 0.5,
+        minimumBlue : 0.05,
+        alpha : 1.0
+    }, {
+        red : 0.9,
+        minimumGreen : 0.6,
+        minimumBlue : 0.01,
+        alpha : 1.0
+    }, {
+        red : 0.8,
+        green : 0.05,
+        minimumBlue : 0.09,
+        alpha : 1.0
+    }, {
+        minimumRed : 1,
+        minimumGreen : 0.05,
+        blue : 0.09,
+        alpha : 1.0
+    }],
+
+    particleSystems : []
+};
+
+var cometOptions = {
+    numberOfParticles : 100.0,
+    iterOffset : 0.003,
+    cartographicStep : 0.0000001,
+    baseRadius : 0.0005,
+
+    colorOptions : [{
+        red : 0.6,
+        green : 0.6,
+        blue : 0.6,
+        alpha : 1.0
+    }, {
+        red : 0.6,
+        green : 0.6,
+        blue : 0.9,
+        alpha : 0.9
+    }, {
+        red : 0.5,
+        green : 0.5,
+        blue : 0.7,
+        alpha : 0.5
+    }, {
+        red : 0.5,
+        green : 0.5,
+        blue : 0.5,
+        alpha : 0.9
+    }],
+
+    particleSystems : []
+};
+
+// Creating the Particles
+function create(usingOptions, newLocation, color, iteration) {
+    var cartesianOrigPosition = Cesium.Cartographic.toCartesian(newLocation);
+    var emitterModelMatrix = Cesium.Matrix4.fromTranslation(cartesianOrigPosition, new Cesium.Matrix4());
+
+    var iterOffset = iteration;
+    var force = function(particle) {
+        var cartesianPosition = particle.position;
+        var downDir = Cesium.Cartesian3.normalize(cartesianPosition, new Cesium.Cartesian3());
+
+        cartesianPosition.x -= downDir.x;
+        cartesianPosition.y -= downDir.y;
+        cartesianPosition.z -= downDir.z;
+
+        var cartographicPosition = Cesium.Cartographic.fromCartesian(cartesianPosition);
+
+        var angle = Cesium.Math.PI * 2.0 * iterOffset / usingOptions.numberOfParticles;
+        iterOffset += usingOptions.iterOffset;
+        cartographicPosition.longitude += Math.cos(angle) * usingOptions.cartographicStep;
+        cartographicPosition.latitude += Math.sin(angle) * usingOptions.cartographicStep;
+
+        cartesianPosition = Cesium.Cartographic.toCartesian(cartographicPosition);
+
+        particle.position = cartesianPosition;
+    };
+
+    var item = viewer.scene.primitives.add(new Cesium.ParticleSystem({
+        image : getImage(),
+        startColor : color,
+        endColor : color.withAlpha(0.0),
+        life : 3.5,
+        speed : 0.00005,
+        width : 15.0,
+        height : 15.0,
+        rate : 30.0,
+        emitter : new Cesium.CircleEmitter(0.1),
+        bursts : [ ],
+        lifeTime : 0.1,
+        forces : [force],
+        modelMatrix : modelMatrix,
+        emitterModelMatrix : emitterModelMatrix
+    }));
+    usingOptions.particleSystems.push(item);
+}
+
+function runParticles(usingOptions) {
```

Comment:
Perhaps add a comment here like "Create several particle systems encircling the base."